packages/idempotency/README.md

```diff
@@ -0,0 +1,83 @@
+# AWS Lambda Powertools for TypeScript

```

Comment:
Hi,

Until we publish the package we can use the README like you suggest since it won't be published anywhere. It's good to have design, class diagrams, etc. near the code & version controlled.

Once we start publishing the package on NPM however I would be inclined to follow the same structure we have for all the others since it's the "landing page" of the package on NPM and customers might arrive to that page without having seen our repo or our docs.